include/groonga/groonga.h

```diff
@@ -1699,6 +1705,20 @@ typedef enum {
 GRN_API const char *
 grn_hook_entry_to_string(grn_hook_entry entry);
 
+/**
+ * \brief Add a hook to the object. If multiple hooks are set for the object,
+ *        they are called in order.
```

Comment:
How about explaining what type of order is used?
```suggestion
 *        they are called in ascending order based on their `offset` values.
```

Reply:
日本語だと「設定してある順番通りに実行します」なのだけれども、英語だとどう表現するのが良いだろうか。
```
they are called in the order of the registered hook list.
```

引数の `offset` は挿入位置を指定してるので、それに従って順番に実行するというのは違和感があるのだけれど、そんなことはない？
配列のindex 0からインクリメントして順番に実行していく感じなんですよね。

Reply:
改めて見返すと、確かに違和感がありますね。
フックリストに登録されてある順番だと、次のような感じが自然かもです。
```
They are called in the order they are registered in the hook list.
```